src/chromecast.cr

```diff
@@ -56,6 +56,20 @@ class Castblock::Chromecast
     end
   end
 
+  def skip_ad(device : Device) : Nil
+    params = HTTP::Params.encode({
+      "uuid" => device.uuid,
+    })
+    response = client.post("/skipad?" + params)
+
+    if !response.status.success?
+      Log.debug &.emit("Error with skipad", status_code: response.status_code, error: response.body)
+      # raise CommandError.new 
+      # We do not want to raise an error based off the response from /skipad as the ad may already be skipped.
+      # The /skipad logic is run 30 times over 60 seconds until the ad is skipped, so subsequent requests as the watcher loops will fail
```

Comment:
If I understand it, when doing the request `/skipad`, go-chromecast will try to skip it for the next 60s? So we must ignore any error as another request can still be running?

Looking at the code in go-chromecast, it looks like the `/skipad` endpoint is sync, so it will not return until it succeed or fail, which could take up to 60s. The handling of a message by castblock is also sync, so while we are waiting for `/skipad` to return, nothing else can be done. Are we sure `/skipad` will return as soon as the ad ends? We do not want to miss a sponsor segment because we are still waiting for `/skipad` to return.

Reply:
I believe the castblock code keeps running - it doesn't wait for a response from what I saw. 
This was a simpler approach instead of waiting, and potentially missing other events. 

I haven't had any issues during my testing.

Reply:
You are right, we do `spawn watch_device(device, @devices[device])`, so it's not sync.